lib/modules/manager/helm-values/extract.ts

```diff
@@ -60,10 +61,15 @@ function findDependencies(
 export function extractPackageFile(content: string): PackageFile | null {
   let parsedContent: Record<string, unknown> | HelmDockerImageDependency;
   try {
+    //This regex checks if type of tag: version is double if yes modify it to string
+    const newContent = content.replace(
+      regEx(/(?<tag>^\s*tag:\s*)(?<ver>[\d.]*$)/gm),
+      '$<tag>"$<ver>"'
+    );
```

Comment:
we need to file an upstream bug and reference it here, so this workaround can be removed.

btw: we use the yaml parser for other managers too, so they have the same issue.

Reply:
https://github.com/nodeca/js-yaml/issues/674